src/View/SSViewer_DataPresenter.php

```diff
@@ -313,7 +341,14 @@ public function __call($name, $arguments)
         if ($val) {
             $obj = $val['obj'];
             if ($name === 'hasValue') {
-                $result = ($obj instanceof ViewableData) ? $obj->exists() : (bool)$obj;
+                // Check if a value exists, e.g. <% if $obj %>
+                if ($obj instanceof ViewableData) {
+                    $result = $obj->exists();
+                } elseif (is_countable($obj)) {
+                    $result = count($obj) > 0;
+                } else {
+                    $result = (bool) $obj;
+                }
```

Comment:
This change allows us to use `<% if $MyArray %>` the same way we'd use `<% if $MyArrayList %>`. It's quite common to rely on this shortcode when checking if a list has values, so it will be intuitive to do it with non-ViewableData iterables as well.

I've made the same change in `ViewableData::hasValue()` for the same reason.